src/Core/functions.php

```diff
@@ -179,7 +179,7 @@ function pj($var)
         }
 
         $template = (PHP_SAPI !== 'cli' && PHP_SAPI !== 'phpdbg') ? '<pre class="pj">%s</pre>' : "\n%s\n\n";
-        printf($template, trim(json_encode($var, JSON_PRETTY_PRINT)));
+        printf($template, trim(json_encode($var, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE)));
```

Comment:
You didnt apply the suggestion as stated. You forgot the JSON_UNESCAPED_SLASHES part.

Reply:
ok